### ISSUE TITLE
Uncaught TypeError: Cannot read property '9' of undefined

### ISSUE THREAD

**Issue by JostCrow** [jeremyramin/terminal-plus#80]
1. run django runserver (python manage.py runserver).
2. defocus the editor.
3. click on the terminal.

It could be that you need to do this multiple times. It just popped up randomly.

**Atom Version**: 1.1.0
**System**: elementary OS
**Thrown From**: [terminal-plus](https://github.com/jeremyramin/terminal-plus) package, v0.14.0

### Stack Trace

Uncaught TypeError: Cannot read property '9' of undefined

```
At /home/jorik/.atom/packages/terminal-plus/node_modules/term.js/src/term.js:5358

TypeError: Cannot read property '9' of undefined
  at Terminal.clearSelectedText (/home/jorik/.atom/packages/terminal-plus/node_modules/term.js/src/term.js:5358:26)
  at HTMLDivElement.<anonymous> (/home/jorik/.atom/packages/terminal-plus/node_modules/term.js/src/term.js:954:12)

```

### Commands

```
  2x -1:57.6.0 editor:select-to-first-character-of-line (atom-text-editor.editor.is-focused)
  2x -1:57.2.0 core:backspace (atom-text-editor.editor.is-focused)
     -1:55.8.0 core:save (atom-text-editor.editor.is-focused)
  3x -1:55.5.0 core:move-up (atom-text-editor.editor.is-focused)
     -1:04.8.0 find-and-replace:show (atom-text-editor.editor.is-focused)
     -1:03.4.0 core:confirm (atom-text-editor.editor.mini.is-focused)
  2x -1:00.3.0 editor:select-to-first-character-of-line (atom-text-editor.editor.is-focused)
  2x -0:59.9.0 core:backspace (atom-text-editor.editor.is-focused)
     -0:59.1.0 core:save (atom-text-editor.editor.is-focused)
  3x -0:46 core:select-up (atom-text-editor.editor.is-focused)
     -0:45.1.0 core:backspace (atom-text-editor.editor.is-focused)
     -0:44.2.0 core:save (atom-text-editor.editor.is-focused)
     -0:32.7.0 editor:newline (atom-text-editor.editor.is-focused)
     -0:32.3.0 editor:select-to-first-character-of-line (atom-text-editor.editor.is-focused)
     -0:32.1.0 core:backspace (atom-text-editor.editor.is-focused)
     -0:31.7.0 core:save (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "themes": [
      "seti-ui-beta",
      "monokai-seti"
    ],
    "openEmptyEditorOnStart": false,
    "autoHideMenuBar": true,
    "destroyEmptyPanes": false
  },
  "terminal-plus": {
    "core": {
      "autoRunCommand": "source env/bin/activate"
    },
    "style": {},
    "toggles": {
      "autoClose": true
    }
  }
}
```

### Installed Packages

```coffee
# User
api-workbench, v0.8.6
apm-publish, v0.2.8
atom-beautify, v0.28.17
atom-jade, v0.3.0
atom-ungit, v0.5.2
autocomplete-python, v0.10.3
autohide-tree-view, v0.25.1
autoupdate-packages, v1.3.0
color-picker, v2.0.13
django-atom, v0.2.4
highlight-line, v0.11.0
hyperclick, v0.0.32
icon-fonts, v1.8.2
imdone-atom, v1.1.55
linter, v1.10.0
linter-pep8, v1.0.1
linter-pylint, v1.0.3
meteor-api, v2.20.0
minimap, v4.16.0
minimap-find-and-replace, v4.3.2
minimap-git-diff, v4.1.8
minimap-highlight-selected, v4.3.1
minimap-linter, v1.0.0
minimap-pigments, v0.1.7
monokai-seti, v0.7.0
nuclide-hg-repository, v0.0.32
pigments, v0.18.1
python-tools, v0.6.5
selection-highlight, v0.1.4
set-syntax, v0.3.0
seti-icons, v0.4.5
seti-ui-beta, v0.1.1
soft-wrap-indicator, v0.7.0
terminal-plus, v0.14.0
tool-bar, v0.1.9
tool-bar-shortcuts, v0.1.5

# Dev
tool-bar-shortcuts, v0.1.5
```

**Comment by jeremyramin** [comment]
@JostCrow I am having trouble reproducing this. Did you happen to select anything at the time?

**Comment by JostCrow** [comment]
@jeremyramin From what I know I did not have anything selected.

It also didn't happen again after that first time.

**Comment by jeremyramin** [comment]
This should be fixed as of v0.14.2

**Comment by JostCrow** [comment]
Thanks. The problem did not occur again.
